locale/en/docs/guides/abi-stability.md

```diff
@@ -55,33 +55,34 @@ within the major version against which it was compiled.
 Demand has arisen for equipping Node.js with an API that results in an ABI that
 remains stable across multiple Node.js major versions. The motivation for
 creating such an API is as follows:
+
 * The JavaScript language has remained compatible with itself since its very
-early days, whereas the ABI of the engine executing the JavaScript code changes
-with every major version of Node.js. This means that applications consisting of
-Node.js packages written entirely in JavaScript need not be recompiled,
-reinstalled, or redeployed as a new major version of Node.js is dropped into
-the production environment in which such applications run. In contrast, if an
-application depends on a package that contains a native addon, the application
-has to be recompiled, reinstalled, and redeployed whenever a new major version
-of Node.js is introduced into the production environment. This disparity
-between Node.js packages containing native addons and those that are written
-entirely in JavaScript has added to the maintenance burden of production
-systems which rely on native addons.
+  early days, whereas the ABI of the engine executing the JavaScript code changes
```

Comment:
Was this flagged in this branch?

Reply:
No, I think it was just the missing space before that was flagged

Reply:
I just wonder how we didn't have any warnings before this PR, that's why I ask :)

Reply:
Yeah, I not sure which rule enforces this. I just updated it since that was the bullet style I saw elsewhere in the docs